doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,1913 @@
+almost every field of science and engineering. It's the universal standard for
+working with numerical data in Python, and it's at the core of the scientific
+Python and PyData ecosystems. NumPy users include everyone from beginning coders
+to experienced researchers doing state-of-the-art scientific and industrial
+research and development. The NumPy API is used extensively in Pandas, SciPy,
+Matplotlib, scikit-learn, scikit-image and most other data science and
+scientific Python packages. 
+
+The NumPy library contains multidimensional array and matrix data structures
+(you'll find more information about this in later sections). It provides
+**ndarray**, a homogeneous n-dimensional array object, with methods to
+efficiently operate on it. NumPy can be used to perform a wide variety of
+mathematical operations on arrays.  It adds powerful data structures to Python
+that guarantee efficient calculations with arrays and matrices and it supplies
+an enormous library of high-level mathematical functions that operate on these
+arrays and matrices. 
+
+`Learn more about NumPy here <https://numpy.org/devdocs/user/whatisnumpy.html>`_!
+
+Installing NumPy 
+----------------
+  
+To install NumPy, we strongly recommend using a scientific Python distribution.
+If you're looking for the full instructions for installing NumPy on your
+operating system, you can `find all of the details here
+<https://www.scipy.org/install.html>`_.
+
+
+  
+If you already have Python, you can install NumPy with::
+
+  conda install numpy
+  
+or ::
+
+  pip install numpy
+  
+If you don't have Python yet, you might want to consider using `Anaconda
+<https://www.anaconda.com/>`_. It's the easiest way to get started. The good
+thing about getting this distribution is the fact that you don’t need to worry
+too much about separately installing NumPy or any of the major packages that
+you’ll be using for your data analyses, like pandas, Scikit-Learn, etc.
+
+You can find all of the installation details in the 
+`Installation <https://www.scipy.org/install.html>`_ section 
+at `SciPy <https://www.scipy.org>`_.
+
+How to import NumPy 
+-------------------
+
+Any time you want to use a package or library in your code, you first need to
+make it accessible. 
+
+In order to start using NumPy and all of the functions available in NumPy,
+you'll need to import it. This can be easily done with this import statement::
+
+  import numpy as np 
+
+(We shorten ``numpy`` to ``np`` in order to save time and also to keep code
+(standardized so that anyone working with your code can easily understand and
+(run it.)
+
+Reading the example code
+------------------------
+
+If you aren't already comfortable with reading tutorials that contain a lot of code, 
+you might not know how to interpret a code block that looks 
+like this::
+
+  >>> a2 = a[np.newaxis, :]
+  >>> a2.shape
+  (1, 6)
+
+If you aren't familiar with this style, it's very easy to understand. 
```

Comment:
It's maybe worth pointing out that:
* This is the style you get when you run `python` on the command line
* If you're using IPython, you might see a different style

Reply:
I'm adding that note!

Reply:
this was done, resolving.